velox/connectors/hive/storage_adapters/abfs/AzureClientProviders.h

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
nit: Let's name this `AzureClientProviderImpl.h/.cpp` to make it clear that the implementations are in this file.